app/_components/SubmitButton.tsx

```
@@ -0,0 +1,27 @@
+export default function SubmitButton({
+	children,
+	pendingLabel,
+	className = "",
+}: SubmitButtonProps) {
+	const { pending } = useFormStatus();
+
+	return (
+		<button
+			disabled={pending}
+			className={`bg-accent-500 px-8 py-4 font-semibold text-primary-800 transition-all hover:bg-accent-600 disabled:cursor-not-allowed disabled:bg-gray-500 disabled:text-gray-300 ${className}`}
+		>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)